### PR TITLE
feat(whitelist): additional tag, attribute, and style attribute support

### PR DESCRIPTION
**Pull request by BlakeStearman**
### Changes
* Reorder `<div>` attributes to alphabetical order to match other declarations
* Add `<del>` tag with attribute `style`
* Adds `aria-label` and `aria-hidden` attributes to `<span>`
* Adds `open` attribute to `<details>`
* Adds the following CSS attributes to CSS whitelist for use in supported `style` attribute:
  * float
  * vertical-align
  * justify-content
  * justify-items
  * justify-self
  * align-items
  * align-self
  * gap
  * grid
  * grid-area
  * grid-auto-columns
  * grid-auto-flow
  * grid-auto-rows
  * grid-column
  * grid-column-end
  * grid-column-start
  * grid-row
  * grid-row-end
  * grid-row-start
  * grid-template
  * grid-template-areas
  * grid-template-columns
  * grid-template-rows

### CODE REVIEW

**Comment by BlakeStearman** [comment]
Follow up coming for test cases.

File: src/index.ts [hunk 1]
```diff
@@ -136,7 +137,29 @@ export class Sanitizer {
...
+    "grid-template": true,
+    "grid-template-areas": true,
+    "grid-template-columns": true,
+    "grid-template-rows": true,
     "line-height": true,
+    "justify-content": true,
+    "justify-items": true,
+    "justify-self": true,
+    "align-items": true,
+    "align-self": true,
```
Comment (dasa) [inline src/index.ts:162]: sort this list?

**Review by dasa — APPROVED** [review]
LGTM. There's a conflict though.

File: src/index.ts (hunk 1 above)
Comment (BlakeStearman) [reply src/index.ts:162]: Good catch, will do 👍

**Comment by BlakeStearman** [comment]
> LGTM. There's a conflict though.

👍 Addressing it locally with test case and review feedback.

File: src/index.ts (hunk 1 above)
Comment (BlakeStearman) [reply src/index.ts]: Applied.

**Review by dasa — APPROVED** [review]
👍🏻 thanks

**Review by ssylvia — APPROVED** [review]

**Pull request merged**